crates/fuel-core/src/executor.rs

```diff
@@ -809,7 +837,6 @@ impl Executor {
         &self,
         tx: &Tx,
         db: &mut Database,
-        block_height: BlockHeight,
```

Comment:
We need to update GraphQL API to return messages with `Spent` status as we do it for `Coin`s.